dask/dataframe/io/csv.py

```diff
@@ -602,6 +602,7 @@ def to_csv(
     mode="wt",
     name_function=None,
     compression=None,
+    compute_kwargs=None,
```

Comment:
Adding a new default kwarg here could potentially break usage for anyone who has been using this method but relying on positional args.  We can either consciously "break" (light breakage) backwards compatibility with a small subset of existing user code or move this new kwarg to just before `**kwargs`.